src/Utilities/Functional.hpp

```diff
@@ -308,6 +308,25 @@ struct Square : Functional<C::arity> {
   }
 };
 
+/// Function for adding two `std::vector`s of `double` component-wise
+struct VectorPlus {
+  std::vector<double> operator()(const std::vector<double>& lhs,
+                                 const std::vector<double>& rhs) const
+      noexcept {
+    ASSERT(lhs.size() == rhs.size(),
+           "Vector sizes in `funcl::VectorPlus` operator do not match. First "
+           "argument "
+           "size: "
```

Comment:
looks like clang-format wasn't smart enough to reflow this string. Please make line 118 `"argument size:"`